apps/omg_utils/test/omg_utils/http_rpc/validators/base_test.exs

```diff
@@ -200,14 +208,36 @@ defmodule OMG.Utils.HttpRPC.Validator.BaseTest do
       assert {:error, {:validation_error, "nint_1", :integer}} == expect(@params, "nint_1", greater: 0)
     end
 
-    test "address should validate both hex value and its length" do
+    test "address should validate both hex value and  length" do
       assert {:ok, @bin_value} == expect(@params, "hex_1", :address)
 
       assert {:error, {:validation_error, "nhex_1", :hex}} == expect(@params, "nhex_1", :address)
 
       assert {:error, {:validation_error, "short", {:length, 20}}} ==
                expect(%{"short" => "0xdeadbeef"}, "short", :address)
     end
+
+    test "signature should validate both hex value and  length" do
+      {:ok, signature_value} = Map.get(@params, "valid_signature") |> Encoding.from_hex()
+      assert {:ok, signature_value} == expect(@params, "valid_signature", :signature)
+
+      assert {:error, {:validation_error, "non_hex_signature", :hex}} ==
+               expect(@params, "non_hex_signature", :signature)
+
+      assert {:error, {:validation_error, "too_short_signature", {:length, 65}}} ==
+               expect(@params, "too_short_signature", :signature)
+    end
+
+    test "hash should validate both hex value and  length" do
+      {:ok, hash_value} = Map.get(@params, "valid_hash") |> Encoding.from_hex()
```

Comment:
```suggestion
      {:ok, hash_value} = @params |> Map.get("valid_hash") |> Encoding.from_hex()
```
 And other occurrences of this. seems like `Credo.Check.Readability.SinglePipe` only catches multiline singlepipes.  @okalouti the codebase code style is expecting us to avoid single pipe occurances.

Reply:
Thanks @InoMurko. Took the chance to clean up pre-existing single pipes in the file as well.